### DISCUSSION TITLE
Artist-producer wants to share relevant access information of a show with culturally Deaf communitiy

### DISCUSSION THREAD

**Discussion by leslieting** [pair-cg/general-contributions#100] [User Stories]
# 1. Motivating Story
_What's the story here? Please describe why you have a particular need._

I am developing an in-person, live multidisciplinary performance called What Brings You In (WBYI), a music-driven, theatrical show. I would like to share with the culturally Deaf community the relevant access information (regarding sound ) incorporated in WBYI.

# 2. Need:
_Now the why is out of the way, what is it you actually need?_

To share relevant access information in WBYI to the culturally Deaf community.

Examples of relevant information ( not a complete list):
Is the production culturally Deaf-led or co-led - Is there a Deaf creator or producer on the creative team
Is the production Deaf-consulted - ex. artistically, for accessibility
Is the venue Deaf-friendly
Is there a Deaf performer
What type of performance is it
Is there an ASL version of the information available in video form

**CQ**: Is this a Deaf-led artistic work?
Answer: yes/no

Proposed terminology: Deaf_creator
Instance of Deaf_creator: John Smith (this would be a personal name) 

Proposed terminology: Deaf_producer
Instance of Deaf_producer: John Smith (personal name) or Xyz Theatre Company (name of a theatre company)

Proposed terminology: Deaf_director
Instance of Deaf_director: John Smith 

**Concepts**:

Culturally Deaf: 
“People who are born into the Deaf Community, and whose first native language is a signed language, not a spoken one, are "culturally Deaf".
 Ref: https://www.signwriting.org/about/questions/quest024.html

“The Deaf community is comprised of culturally Deaf people in the core of the community who use a sign language (e.g. American Sign Language or Langue des Signes Quebecois) and appreciate their heritage, history, literature, and culture. The Deaf community is also comprised of other individuals who use the language and have an attitude that makes them an accepted part of the community though they may not be in the core of the community.” 
Ref: https://deafculturecentre.ca/what-is-deaf-culture/

Creator(s): the person who had the initial main idea for the performance/event/experience, may be more than one person
Producer(s): the person who turns the idea into an actual experience/event for others
Deaf-led: the creator and/or producer (could be the same or multiple people) is culturally Deaf and so the performance is informed by the Deaf cultural experience

**CQ**: Was there culturally Deaf consultation involved in the artistic production?
Answer: Yes/no

Proposed terminology: Deaf_community_consultation_type 
Instances of Deaf_community_consultation_type: 

- Artistic
- Outreach / Marketing / Communication
- Patron support: Etiquette, Front of House
- ?

Proposed terminology: Deaf_community_consultant_name
Instances of Deaf_community_consultant_name: 

- John Smith, Deaf consultant
- John Smith, Deaf artist

Proposed terminology: Deaf_community_consultant_title
Instances of Deaf_community_consultant_title: 

- Deaf consultant; 
- Deaf artist
- Accessibility consultant

**CQ**: Is this specific presentation Deaf-friendly?
Answer: Yes (hypothetical)

Possible examples of ways a venue would be Deaf-friendly (need confirmation these are actual) access contact, the front of house has been trained (ex. pen and paper ready, knows basic Deaf etiquette) , there is an ASL interpreter at the box office, captioning available

Proposed terminology: show_format
Instance of show_format: 

- In-person: Indoor, outdoor, hybrid
- Digital: Software App, Online video (live stream), Online video (pre-recorded)

Proposed terminology: show_date
Instance of show_date: 

- Specific Date
- Specific time
- on-demand

Proposed terminology: show_location
Instance of show_location: 

- Venue name
- Venue address

Proposed terminology: Deaf_friendly_tech
Instance of Deaf_friendly_tech: 

- closed captioning or subtitles
- ASL interpreter at box office
- access contact

Proposed terminology: Deaf_friendly_cultural_experience
Instance of Deaf_friendly_cultural_experience: 

- ASL-first communication practices: Eye contact, Clear/Direct speech, Pen and Paper
- ASL interpreter at box office
- access contact

**Concepts:** 
Ref -  deaffriendly.com: 
Deaf-friendliness can mean different things to different people. Some perceive it emotionally, such as effortless empathy from hearing people who intuitively know how to meet our needs. Others see it as a technologically, such as an apartment retrofitted with blinking strobe lights – or a theatre with subtitle glasses or Rear Window® captioning.
As a bare minimum, deaf-friendliness includes eye contact, clear speech, willingness to use pen and paper, and patience. Whatever individual perceptions are, deaf-friendliness is not something businesses can afford to skimp on.

**CQ**: Is there a Deaf performer?
Answer: Yes/No
Proposed terminology: Deaf_performer_role
Instance of Deaf performer: ex. actor, dancer, musician, performer

Proposed terminology: Deaf_performer_name
Instance of Deaf_performer_name: Landon Krentz

**CQ:** Is there a Deaf artistic collaborator?
Answer: Yes/No
Proposed terminology: Deaf_artistic_collaborator
Instance of Deaf_artistic_collaborator: ex. choreographer, lighting designer, director, set designer

**CQ:** What type of performance is it?
Answer: music-driven theatrical performance
Proposed terminology: multi-disciplinary
Instance of multi-disciplinary: music-driven theatrical performance

# 3. Action:
_What is an action that can be taken towards fulfilling that need?_
To publish the relevant information (defined under Needs) through channels x, y, z, (to be defined)

# 4. Desired Outcome:
_What is the desired outcome?_

 
To make the relevant information available and known about the show to Deaf communities. 

# 5. Perspective:
_From what perspective is this user story coming from?_

Hearing creator/producer/performer of What Brings You In. 

# 6. Stakeholder Group:
_Which stakeholder groups are impacted by this User Story?_

Deaf communities
The channels Deaf communities 
Producer/marketers of WBYI

# 7. Additional info:
_Here's more information._

https://www.leslieting.com/what-brings-you-in

Show background: 

What Brings You In (WBYI) is the latest multi-disciplinary work by Leslie Ting Productions. Since 2007, Leslie has periodically gone to therapy, always finding the process strange, but personally necessary. WBYI was born out of a curiosity about what compelled others to go to therapy. Her research led to WBYI’s broader theme of self-understanding in relation to others. Initial development included extensive hands-on research and 40 field interviews with psychologists, counselors, alternative therapists and lay people. A one-hour, music-driven performance piece created by Leslie Ting Productions (LTP) and performed/directed by Leslie (violin), Germaine Liu (percussion) and Elyshia Poirier (live video), WBYI explores the concept of “what brings us in” - to a concert hall, a communal space, our deeper nature. 

The in-person experience of WBYI spans artistic disciplines of live video art, immersive sound and lighting design to express the paradox of discovering oneself through the experience of connecting with others. Comprised of six vignettes, each centers around a musical performance with integrated staging, lighting and sound spatialization, all animating the underlying WBYI thematic material of self-exploration. Clips of the one-on-one field research interviews are weaved throughout the vignettes and contextualize the overall arc of the piece. 

# 8. Competency Questions:

See 2. Need, above

**Comment by marPAIR** [comment]
Hi Leslie, some thoughts for our call below. In general, I think there is a shift in tone from the "Motivation Story", which is much more narrative, into the rest of the user story, that needs to happen. 

1. I think the "Needs" should be framed more as about making relevant information about your show available to the differently-abled groups that you have identified. Your current need as articulated is not wrong, per se, but framing it this way will make way for us to drill down into the details what qualifies as "relevant information", and result in a concrete need against which we can develop competency questions.

2. In terms of "Action", might this be informed by the channels through which information is accessed by the differently-abled groups? We want to phrase this part of the story in a way that reads like this: "I will publish the 'relevant information' (defined under Needs) through channels x, y z (to be identified)". Maybe we can reduce the scope of this user story to just one specific group, such as Deaf, and get Landon to help us shape it?

3. What you described in "Desired Outcome" is certainly what we need to do, in order to build out this user story! :) 

I'll switch gears and brainstorm a few competency questions here for discussion:

1. Is this a Deaf-led or Deaf-friendly performance?
2. Does this show feature a Deaf performer?
3. Does this show feature a blind or partially-sighted performer?
4. What type of performance is it?
5. Is there a contact for access needs for this performance?

**Reply by leslieting** [reply]
HI Margaret - yes, I was definitely finding "differently abled" groups to be much too big a category! 
To your first point on Needs - I think I'm trying to take a step back into wanting to know what information each group would like to know first. I wonder/assume there are gaps in my knowledge as an able bodied person, and wonder if there is more. In which case, yes, with Landon in the group, we could just ask him and do the research ahead of time and narrow down this story, rather than make the research itself _the_ story!

**Comment by leslieting** [comment]
See Google doc with additional questions and comments on this user story here: https://docs.google.com/document/d/112zZF58wdVrVOxQVdETVzjW-ZWVBWb4ZAm3x-zdw284/edit?usp=sharing
